### DISCUSSION TITLE
Deprecate old projects

### DISCUSSION THREAD

**Discussion by KleinerHacker** [openupm/openupm#3672] [General]
Hello,

sometimes I merge already published projects in OpenUPM to one project. In this case I want to mark the old projects as deprecated. 
Is there a way to do this? Or to hide the old packages?

Thanks.

**Comment by favoyang** [comment]
The Unity Package Manager doesn't support deprecated packages/versions.

- https://forum.unity.com/threads/deprecating-or-unpublishing-packages.917486/
- https://forum.unity.com/threads/how-can-i-deprecate-a-custom-package.877357/

But you can update your GitHub readme and leave a note to point to your new project, and perhaps mark the GitHub project as archived.

Deleting old packages is not an encouraged practice for a published package.

**Reply by KleinerHacker** [reply]
There are news with this theme? Is is supported now? In the thread I read that is a possible future feature.

**Reply by favoyang** [reply]
No news
